contributing.md

```diff
@@ -0,0 +1,38 @@
+# Contributing
```

Comment:
The README and the contents of this file are very similar. I think we can just get rid of this file all together for now.